src/main/resources/swagger.api/schemas/itemContributionOptionsConfigurationDTO.json

```diff
@@ -0,0 +1,48 @@
+{
+  "description": "Item Contribution Options",
+  "type": "object",
+  "properties": {
+    "centralServerId": {
```

Comment:
Once you define centralServerId in each and every URL it becomes unnecessary here and can be removed